packages/admin-ui/tailwind.config.js

```diff
@@ -20,48 +33,22 @@ module.exports = {
             }
         },
         extend: {
```

Comment:
Should we add our tokens and override the default class names? 
As we agreed not to use Tailwind's default class names, I would remove them to highlight the missing token definitions.

Reply:
Sounds good.

Reply:
Moved everything to overrides (out of the `extend` section).